runtime/autoload/remote/host.vim

```diff
@@ -207,7 +210,7 @@ endfunction
 " Registration of standard hosts
 
 " Python/Python3
-call remote#host#Register('python', '*.py',
+call remote#host#Register('python', '*',
       \ function('provider#pythonx#Require'))
-call remote#host#Register('python3', '*.py',
+call remote#host#Register('python3', '*',
```

Comment:
`unlet! specs` is very real. The others are real as well if we don't want misleading messages.